src/js/containers/covid19/homepage/CovidHighlights.jsx

```diff
@@ -65,7 +70,7 @@ export class CovidHighlights extends React.Component {
         this.handleResizeWindow();
         return this.fetchDefCodes()
             .then(() => (
-                Promise.all([this.fetchHighlights(), this.fetchTotals()])
+                Promise.all([this.fetchHighlights(), this.fetchTotals(), this.fetchSubmissionMonth()])
```

Comment:
We might actually want to make this resolve before we fetch totals and highlights to give the effect of the `as of data` being ready before the total.